docs/user-guide-state-reporting.md

```diff
@@ -14,6 +14,28 @@ on [local cluster](deployment-local-cluster.md) or your
 
 The operator will periodically update the reported state of node interfaces.
 
+We can set the period of update time in seconds in config map in variable 
```

Comment:
Please add a new section "Additional Configuration" to the bottom of the file with this text